### PR TITLE
updated rules.mk and default keymap of Wonderland for VIA support

### PR DESCRIPTION
**Pull request by ThePanduuh**
## Description

Added VIA_ENABLE to rules.mk to allow VIA port. To keep VELOCIKEY_ENABLE=yes had to use LTO_ENABLE=yes to shrink file size. Added NKRO since it didn't take up any additional space and can be toggled using LSHIFT+RSHIFT+N on this PCB. Also modified the default keymap to add a KC_TRANS under the MO(1) key as it would cause layer to be toggled permanently (more for ease of use on end user).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/maartenwut/wonderland/rules.mk [hunk 1]
```diff
@@ -28,3 +28,5 @@ UNICODE_ENABLE = no 		# Unicode
 BLUETOOTH_ENABLE = no # Enable Bluetooth with the Adafruit EZ-Key HID
 AUTO_SHIFT_ENABLE = no
 VELOCIKEY_ENABLE = yes
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```
Comment (zvecr) [inline keyboards/maartenwut/wonderland/rules.mk:32]: The convention is to add a `via` keymap, and not enable for all keymaps
```suggestion
```

Comment (ThePanduuh) [reply keyboards/maartenwut/wonderland/rules.mk:32]: ah, sorry about that! I will make the changes and make a via keymap folder with rules.mk and the modified default keymap. Thank you!

File: keyboards/maartenwut/wonderland/keymaps/default/keymap.c [hunk 2]
```diff
@@ -18,7 +18,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  RGB_TOG, VLK_TOG,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,  KC_F12, XXXXXXX, RESET, \
  RGB_MOD, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, \
 RGB_RMOD, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,   KC_UP, XXXXXXX, \
-          KC_LSFT, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, XXXXXXX, XXXXXXX, XXXXXXX, KC_LEFT, KC_DOWN,KC_RIGHT, XXXXXXX, \
+          KC_LSFT, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, XXXXXXX, XXXXXXX, XXXXXXX, KC_LEFT, KC_DOWN,KC_RIGHT, KC_NO, \
```
Comment (fauxpark) [inline keyboards/maartenwut/wonderland/keymaps/default/keymap.c:21]:
```suggestion
          KC_LSFT, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, XXXXXXX, XXXXXXX, XXXXXXX, KC_LEFT, KC_DOWN,KC_RIGHT, XXXXXXX, \
```

Comment (ThePanduuh) [reply keyboards/maartenwut/wonderland/keymaps/default/keymap.c:21]: TIL XXXXXXX == KC_NO and _______ == KC_TRNS, fixing this now.

File: keyboards/maartenwut/wonderland/keymaps/via/rules.mk [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+RGBLIGHT_ENABLE = yes   # Enable keyboard underlight functionality
+BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
+MIDI_ENABLE = no 		# MIDI controls
+AUDIO_ENABLE = no
+UNICODE_ENABLE = no 		# Unicode
+BLUETOOTH_ENABLE = no # Enable Bluetooth with the Adafruit EZ-Key HID
+AUTO_SHIFT_ENABLE = no
+VELOCIKEY_ENABLE = yes
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```
Comment (zvecr) [inline keyboards/maartenwut/wonderland/keymaps/via/rules.mk:32]: This file should only contain overrides,
```suggestion
VIA_ENABLE = yes
LTO_ENABLE = yes
```

Comment (ThePanduuh) [reply keyboards/maartenwut/wonderland/keymaps/via/rules.mk:32]: More things I didn't know needed to exist, thank you for correcting.

**Comment by drashna** [comment]
@Maartenwut any issues with this?

File: keyboards/maartenwut/wonderland/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+	[3] = LAYOUT(_______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______)
```
Comment (noroadsleft) [inline keyboards/maartenwut/wonderland/keymaps/via/keymap.c:8]: Making this more readable.

```suggestion
	[0] = LAYOUT(
        KC_ESC,  KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_DEL,  KC_BSPC,
        KC_PGUP, KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
        KC_PGDN, KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
                 KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),
                 KC_LCTL,          KC_LAPO,          KC_BSPC, KC_LGUI, KC_SPC,                    KC_RAPC,                            KC_RCTL
    ),
	[1] = LAYOUT(
        RGB_TOG,  VLK_TOG, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_NO,   RESET,
        RGB_MOD,  KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,
        RGB_RMOD, KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_UP,            KC_NO,
                  _______, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, KC_NO,   KC_NO,   KC_NO,   KC_LEFT, KC_DOWN, KC_RGHT, _______,
                  _______,          KC_LALT,          _______, _______, _______,                   KC_RALT,                            _______
    ),
	[2] = LAYOUT(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
                 _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
                 _______,          _______,          _______, _______, _______,                   _______,                            _______
    ),
	[3] = LAYOUT(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
                 _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
                 _______,          _______,          _______, _______, _______,                   _______,                            _______
    )
```

**Comment by Maartenwut** [comment]
You got my approval!

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**